hi-ml/src/health_ml/lightning_container.py

```diff
@@ -178,6 +178,8 @@ def get_hyperdrive_config(self) -> Optional[HyperDriveConfig]:
             return self.get_crossval_hyperdrive_config()
         if self.hyperdrive:
             return self.get_parameter_tuning_config(ScriptRunConfig(source_directory=""))
+        if self.different_seeds > 0:
+            return self.get_different_seeds_hyperdrive_config()
```

Comment:
docstring to be updated to include this option

Reply:
good point, yes.